bittensor/_neuron/text/advanced_server/sample_config.txt

```diff
@@ -0,0 +1,54 @@
+axon.backward_timeout: 20
```

Comment:
Also, it might be good to have all of the config files in the same folder. Maybe call it bittensor/configs/....

Reply:
done